infrastructure/validators/build-a-community.md

```diff
@@ -67,13 +65,13 @@ corresponding field in their identity so nominators can find it easily.
 
 ## Transparency & Establishing Trust
 
-Considering the risks involved for both Validators and Nominators, establishing trust is one of the
+Considering the risks involved for both validators and nominators, establishing trust is one of the
 most essential factors in running quality validator services.
 
 ### Self Stake
 
 Validators should have skin in the game in their operations in the form of a stake that is
-self-bonded to their validator stash. [Slashing](../learn/learn-offenses.md) applies to the total
+self-bonded to their validator stash. [Slashing]() applies to the total
 stake of a Validator, therefore having a high self-stake shows confidence in the operations. This
 helps show commitment from the Validator as they have skin in the game and can be penalized for
 negative actions or poor maintenance. mess up. Having very little self-stake can signal to
```

Comment:
"This helps show commitment from the Validator as they have skin in the game and can be penalized for
negative actions or poor maintenance. **mess up.** Having..."

Note: The 'mess up.' is out of place and does not fit into the sentence. cc @CrackTheCode016